FAQ.md

```diff
@@ -115,12 +115,18 @@ $ rg --generate complete-fish > "$dir/rg.fish"
 
 For **zsh**:
 
-```
+```zsh
 $ dir="$HOME/.zsh-complete"
 $ mkdir -p "$dir"
 $ rg --generate complete-zsh > "$dir/_rg"
 ```
```

Comment:
I can do that.

...orr, we could suggest the easier way I'm adding in this commit.

> If you installed ripgrep with a package manager, you should have shell completions automatically (Brew users see [here](https://docs.brew.sh/Shell-Completion))
> However, if completions aren't working, then add to your `zshrc`
> ```zsh
> source <(rg --generate=complete-zsh)
> ```

I like the idea of hoping their package manager does the right thing, but if doesn't then suggesting a simple one-liner instead of a multi-step process of creating directories, files, and adding things to your `fpath`.  On my machine (granted, it's over $1k) it only adds 4ms of overhead which is acceptable imo.